avocado_vt/test.py

```diff
@@ -404,7 +403,8 @@ def _runTest(self):
                         raise error.TestWarn("funcatexit failed with: %s" %
                                              error_message)
 
-                except Exception:
+                except:  # nopep8 Old-style exceptions are not inherited from Exception()
```

Comment:
Do we have the old-style exceptions in code?

Reply:
There are/were some in `tp-qemu` and we can't say anything about user-test-repos. So yes, it's useful...